### PR TITLE
Bug 1335496 - Use the new Treestatus api

### PR DESCRIPTION
**Pull request by KWierso**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/2120)

### CODE REVIEW

File: tests/ui/unit/controllers/jobs.tests.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ describe('JobsCtrl', function(){
             getJSONFixture('job_list/job_2.json')
         );
 
-        $httpBackend.whenGET('https://api.pub.build.mozilla.org/treestatus/trees/mozilla-central').respond(
+        $httpBackend.whenGET('http://treestatus.mozilla-releng.net/trees/mozilla-central').respond(
```
Comment (garbas) [inline tests/ui/unit/controllers/jobs.tests.js]: it should be ``https``.

File: tests/ui/unit/models/resultsets.tests.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ describe('ThResultSetStore', function(){
 
 
 
-        $httpBackend.whenGET('https://api.pub.build.mozilla.org/treestatus/trees/mozilla-central').respond(
+        $httpBackend.whenGET('http://treestatus.mozilla-releng.net/trees/mozilla-central').respond(
```
Comment (garbas) [inline tests/ui/unit/models/resultsets.tests.js]: it should be https.

File: ui/js/services/treestatus.js [hunk 3]
```diff
@@ -3,7 +3,7 @@
 treeherder.factory('treeStatus', [
     '$http',
     function($http) {
-        var urlBase = "https://api.pub.build.mozilla.org/treestatus/trees/";
+        var urlBase = "http://treestatus.mozilla-releng.net/trees/";
```
Comment (garbas) [inline ui/js/services/treestatus.js]: it should be https.

**Comment by KWierso** [comment]
Fixed the http -> https stuff. Not sure how that happened since I copied the URLs directly from my browser, and it redirects to https automatically...

**Pull request merged**